spec/models/like_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  let(:user) { create(:user) }
+  let(:post) { create(:Post) }
+
+  describe 'methods' do
+    it { should belong_to(:author).class_name('User') }
+    it { should belong_to(:post).class_name('Post') }
+  end
+end
```

Comment:
- [ ] kindly add the test for the method ` update_likes_counter`

Reply:
> * [ ]  kindly add the test for the method ` update_likes_counter`

- [ ] As suggested by the previous reviewer, we need to test the  `update_likes_counter` method inside the `Like` model. Please add the test for the method.